sherpa/astro/ui/utils.py

```diff
@@ -12105,6 +12167,78 @@ def plot_arf(self, id=None, resp_id=None, replot=False, overplot=False,
         self._plot(plotobj, overplot=overplot, clearwindow=clearwindow,
                    **kwargs)
 
+
+    def plot_rmf(self, id=None, resp_id=None, replot=False, overplot=False,
+                 clearwindow=True, **kwargs):
+        """Plot the RMF associated with a data set.
+
+        Display the energy redistribution from the RMF
+        component of a PHA data set.
+
+        Parameters
+        ----------
+        id : int or str, optional
+           The data set with an ARF. If not given then the default
+           identifier is used, as returned by `get_default_id`.
+        resp_id : int or str, optional
+           Which ARF to use in the case that multiple ARFs are
+           associated with a data set. The default is ``None``,
+           which means the first one.
+        replot : bool, optional
+           Set to ``True`` to use the values calculated by the last
+           call to `plot_data`. The default is ``False``.
+        overplot : bool, optional
+           If ``True`` then add the data to an existing plot, otherwise
+           create a new plot. The default is ``False``.
+        clearwindow : bool, optional
+           Should the existing plot area be cleared before creating this
+           new plot (e.g. for multi-panel plots)?
+
+        Raises
+        ------
+        sherpa.utils.err.ArgumentErr
+           If the data set does not contain PHA data.
+
+        See Also
+        --------
+        get_rmf_plot : Return the data used by plot_rmf.
+        plot : Create one or more plot types.
+
+        Examples
+        --------
+
+        Plot the RMF for the default data set:
+
+        >>> plot_rmf()
+
+        Plot the RMF from data set 1 and overplot
+        the RMF from data set 2:
+
+        >>> plot_rmf(1)
+        >>> plot_rmf(2, overplot=True)
+
+        Plot the RMFs labelled "rmf1" and "rmf2" for the
+        "src" data set:
+
+        >>> plot_arf("src", "rmf1")
+        >>> plot_arf("src", "rmf2", overplot=True)
```

Comment:
```suggestion
        >>> plot_rmf("src", "rmf1")
        >>> plot_rmf("src", "rmf2", overplot=True)
```

although we normally just use 1, 2, ... when we have multiple responses (you should be able to use strings, I just haven't checked as there's a nagging worry there's some internal location where we assume an integer key for the responses, but this is all assumption)

Reply:
That is - as you see - copied from plot_arf, which uses "arf1" and "arf2" labels... so if it's questionable here, it's questionable there.